src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
 body {
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card {
+  padding: 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: 200px;
+
+  &__img {
+    margin: 0 auto;
+  }
+
+  &__item {
+    display: flex;
+    margin-top: 16px;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    color: #060b35;
+    line-height: 18px;
+  }
+
+}
+// після того як додаю card__code у .code усе ламається
+.card__code {
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  color: #060b35;
+  font-size: 10px;
+  font-weight: 400;
+  width: 100%;
+  margin: 0 0 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
Here you also can try creating structure: 
.stars {
  &__star{
    &:nth-child...
  }
}